src/document.h

```diff
@@ -496,6 +480,23 @@ struct Document {
         return;
     }
 
+    #ifdef __WXGTK__

```

Comment:
why did you split it up by duplicating the code? the old code was better

Reply:
The return types differ between __WXGTK__ and other platforms.

Reply:
Ok I changed that. But I need to set the return type and differentiate between _WXGTK_ and others as also within the function body, there is a second differentiation then.